dipy/workflows/tests/test_align.py

```diff
@@ -20,12 +29,105 @@ def test_reslice():
-        
+
+
+def test_image_registration():
+
+    with TemporaryDirectory() as temp_out_dir:
+        static, moving, static_g2w, moving_g2w, smask, mmask, M\
+            = setup_random_transform(transform=regtransforms[('AFFINE', 3)],
+                                     rfactor=0.1)
+
+        save_nifti(pjoin(temp_out_dir, 'b0.nii.gz'), data=static,
+                   affine=static_g2w)
+        save_nifti(pjoin(temp_out_dir, 't1.nii.gz'), data=moving,
+                   affine=moving_g2w)
+
+        static_image_file = pjoin(temp_out_dir, 'b0.nii.gz')
+        moving_image_file = pjoin(temp_out_dir, 't1.nii.gz')
+
+        out_moved = pjoin(temp_out_dir, "com_moved.nii.gz")
+        out_affine = pjoin(temp_out_dir, "com_affine.txt")
+
+        image_registeration_flow = ImageRegistrationFlow()
+
+        image_registeration_flow.run(static_image_file, moving_image_file,
+                                     transform='com', out_dir=temp_out_dir,
+                                     out_moved=out_moved,
+                                     out_affine=out_affine,
+                                     level_iters=[100, 10, 1])
+
+        npt.assert_equal(os.path.exists(out_moved), True)
+        npt.assert_equal(os.path.exists(out_affine), True)
+
+        out_moved = pjoin(temp_out_dir, "trans_moved.nii.gz")
+        out_affine = pjoin(temp_out_dir, "trans_affine.txt")
+
+        image_registeration_flow.run(static_image_file, moving_image_file,
+                                     transform='trans', out_dir=temp_out_dir,
+                                     out_moved=out_moved,
+                                     out_affine=out_affine,
+                                     level_iters=[100, 10, 1])
+
+        npt.assert_equal(os.path.exists(out_moved), True)
+        npt.assert_equal(os.path.exists(out_affine), True)
+
+        out_moved = pjoin(temp_out_dir, "rigid_moved.nii.gz")
+        out_affine = pjoin(temp_out_dir, "rigid_affine.txt")
+
+        image_registeration_flow.run(static_image_file, moving_image_file,
+                                     transform='rigid', out_dir=temp_out_dir,
+                                     out_moved=out_moved,
+                                     out_affine=out_affine,
+                                     level_iters=[100, 10, 1])
+
+        npt.assert_equal(os.path.exists(out_moved), True)
+        npt.assert_equal(os.path.exists(out_affine), True)
+
+        out_moved = pjoin(temp_out_dir, "affine_moved.nii.gz")
+        out_affine = pjoin(temp_out_dir, "affine_affine.txt")
+
+        image_registeration_flow.run(static_image_file, moving_image_file,
+                                     transform='affine', out_dir=temp_out_dir,
+                                     out_moved=out_moved,
+                                     out_affine=out_affine,
+                                     level_iters=[100, 10, 1])
+
+        npt.assert_equal(os.path.exists(out_moved), True)
```

Comment:
As advised in another review, simply use the `assert` statement when writing `npt.assert_equal(..., True)`